bucket/robo3t.json

```diff
@@ -1,27 +1,22 @@
 {
-    "version": "1.4.2",
+    "version": "1.4.3",
     "description": "The free lightweight GUI for MongoDB enthusiasts",
     "homepage": "https://robomongo.org",
     "license": "GPL-3.0-only",
-    "architecture": {
-        "64bit": {
-            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.2-windows-x86_64-8650949.zip",
-            "hash": "6f4123c7a1d5ee7121d5f1af942e48140bb6e40008040d4caa17878324e34050",
-            "extract_dir": "robo3t-1.4.2-windows-x86_64-8650949"
-        }
-    },
+    "url": "https://github.com/Studio3T/robomongo/releases/download/v1.4.3/robo3t-1.4.3-windows-x86_64-48f7dfde.zip",
+    "extract_dir": "robo3t-1.4.3-windows-x86_64-48f7dfde",
+    "hash": "15c0d1d5751d68023fcca0f9e0db14dc69ea4607c2a4c9645df10b380c1653dd",
     "shortcuts": [
         [
             "robo3t.exe",
             "Robo 3T"
         ]
     ],
+    "checkver": {
+        "url": "https://github.com/Studio3T/robomongo/releases",
+        "regex": "tag/v([\\w.-]+)"
+    },
     "autoupdate": {
-        "architecture": {
```

Comment:
Unless the tool is 32 bit only, for new updates we prefer retaining architecture and 64bit for 64bit apps to minimize inconvenience for 32 bit users

So please use the old format

But newer GitHub URLs should be obviously retained